architecture/core_objects/builds.adoc

```diff
@@ -125,14 +62,14 @@ accidental or intentional abuses of the build system.
 Operational security:: Building an arbitrary *_Dockerfile_* exposes the host
 system to root privilege escalation. This can be exploited by a malicious user
 because the entire docker build process is run as a user with docker privileges.
-STI restricts the operations performed as a root user, and can run the scripts
+S2I restricts the operations performed as a root user, and can run the scripts
 as an individual user.
```

Comment:
I know this was already here... but "**individual** user" doesn't sound like the right term to me. Shouldn't it be...

"S2I restricts the operations performed as root, and can run the scripts as a non-root user."
or
"S2I restricts the operations performed as root, and can run the scripts as an unprivileged user."